src/main/scala/uk/gov/hmrc/hmrcfrontend/config/AccountMenuConfig.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.config
+
+import play.api.Configuration
+import javax.inject.Inject
+
+class AccountMenuConfig @Inject() (configuration: Configuration) {
+
+  private val perTaxFrontendHost: Option[String] =
+    configuration.getOptional[String]("pta-account-menu.pertax-frontend.host")
+
+  private def baseUrl: String = perTaxFrontendHost.getOrElse("")
+
+  val accountHomeHref: String   =
+    s"$baseUrl${configuration.get[String]("pta-account-menu.accountHomePath")}"
+  val messagesHref: String      =
+    s"$baseUrl${configuration.get[String]("pta-account-menu.messagesPath")}"
+  val checkProgressHref: String =
+    s"$baseUrl${configuration.get[String]("pta-account-menu.checkProgressPath")}"
```

Comment:
@kyle-bowden I think this should have a different base url, it's only locally that they're not the same, but this might be quite an important component to be able to have working locally. This is tracking-frontend I think, @Muhammad-Ali-Ahmad might be able to confirm

Reply:
I have introduced the `pertax-frontend.host` configuration, so that this can be set explicitly if needed.

Reply:
Ah, @kyle-bowden sorry, pertax-frontend makes sense to me, but not all of these urls are provided by pertax-frontend, this one (check progress) is from https://github.com/hmrc/tracking-frontend I think (https://github.com/hmrc/tracking-frontend/blob/main/conf/prod.routes#L2)

Reply:
So i have added config to override the hosts for individual links, by default it will use the `pta-account-menu.pertax-frontend.host` but they can be overridden in each service link configuration as in the example below.

`pta-account-menu {
  accountHome = {
    host = "http://localhost:12345"
  }

  messages = {
    host = "http://localhost:12346"
  }

  checkProgress = {
    host = "http://localhost:12347"
  }

  yourProfile = {
    host = "http://localhost:12348"
  }
}`